yapapi/golem.py

```diff
@@ -59,6 +59,24 @@ class Golem(_Engine):
     at any given time.
     """
 
+    async def start(self) -> None:
```

Comment:
weren't these methods supposed to be implemented on the `_Engine`?

Reply:
1. All logic is on `_Engine`.
2. Those are public methods. 
--> I think this is 100% in line with the Twist discussion